### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by BangKuntet** [Schweinepriester/github-profile-achievements#261]
can you make tutorial to get all badge
newbie is hard to understand to get badge

**Comment by Schweinepriester** [comment]
Yep, I intend to eventually expand [this section](https://github.com/Schweinepriester/github-profile-achievements#specifics-on-how-to-achieve) to cover that.
But it looks very much like GitHub is still tinkering with the criteria, hence I haven't made the effort.

**Comment by RUPAAK** [comment]
How can i get the global brain achievement. Can you please explain

**Comment by BangKuntet** [comment]
you just ask and answer with diferent accounts in github comunity

Pada tanggal Kam, 11 Agu 2022 02:48, RUPAK ***@***.***>
menulis:

**Comment by jadehamel** [comment]
Could it be something like a Slack workspace for this kind of request?

**Comment by fullstack-webdev** [comment]
It's first time for me to be here.
